packages/web/src/pages/commanders/commandersList.tsx

```diff
@@ -19,7 +20,8 @@ export const CommandersList = () => {
   }>();
 
   useEffect(() => {
-    document.title = `${commanderBase} - ${capitalize(race)}`;
```

Comment:
Is there any reason for this change?

Reply:
I had issues with the capitalize during the migration. If the original line is kept, I would face this error.
<img width="824" alt="Screenshot 2024-10-19 at 4 45 50 PM" src="https://github.com/user-attachments/assets/6d18a245-e633-4f7f-93cb-ab1e8fe5b46f">

Reply:
I see, you can fix this by passing there empty string like this ${capitalize(race || "")}